toolchain/sem_ir/name_scope.h

```diff
@@ -18,13 +18,104 @@ enum class AccessKind : int8_t {
   Private,
 };
 
+// Represents the result of a name lookup.
+// A lookup result can be:
+// * Successful returning a valid `InstId`,
```

Comment:
The comma at the end looks like a typo; but a couple different suggestions related to phrasing:

1. Comma after successful to adjust grammar slightly:

```suggestion
// * Successful, returning a valid `InstId`.
```

2. Use "when xyz" similar to below:

```suggestion
// * Successful when returning a valid `InstId`.
```

Reply:
Done.